v2/bundle/x509bundle/bundle.go

```diff
@@ -1,65 +1,175 @@
 package x509bundle
 
 import (
+	"bytes"
 	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"fmt"
 	"io"
+	"io/ioutil"
+	"sync"
 
 	"github.com/spiffe/go-spiffe/v2/spiffeid"
 )
 
+const certType string = "CERTIFICATE"
+
 // Bundle is a collection of trusted public key material for a trust domain.
 type Bundle struct {
+	trustDomain spiffeid.TrustDomain
+	roots       []*x509.Certificate
+	rootsMtx    sync.RWMutex
 }
 
 // New creates a new bundle
 func New(trustDomain spiffeid.TrustDomain) *Bundle {
-	panic("not implemented")
+	return &Bundle{
+		trustDomain: trustDomain,
+		roots:       []*x509.Certificate{},
+	}
 }
 
 // Load loads a Bundle from a file on disk.
 func Load(trustDomain spiffeid.TrustDomain, path string) (*Bundle, error) {
-	panic("not implemented")
+	fileBytes, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, fmt.Errorf("unable to load X.509 bundle file: %w", err)
+	}
+
+	return Parse(trustDomain, fileBytes)
 }
 
 // Read decodes a bundle from a reader.
 func Read(trustDomain spiffeid.TrustDomain, r io.Reader) (*Bundle, error) {
-	panic("not implemented")
+	var b bytes.Buffer
```

Comment:
nit: `ioutil.ReadAll` is probably a better fit here since we don't really need a buffer (just the bytes)